### ISSUE TITLE
Cannot open videos from feed - is this normal? 

### ISSUE THREAD

**Issue by IngoMontaigne** [fetlife/android#550]
I recently subscribed, and cannot open "loved" videos from my feed. Is this normal?  When I try in my phone browser it works fine.

**Comment by kistitan** [comment]
Hi @IngoMontaigne ,

No, it is not normal, if you are a subscriber, you should be able to open videos.

Would you mind, trying the following:
In the FetLife App's menu open:

Android Settings -> Profile Settings -> Clear local User Data -> (After date you will need to log in again)

Please let me know if it changed the bahviour.

Thanks a lot and sorry for the inconveniences

**Comment by IngoMontaigne** [comment]
Nope, no change. I also cleared local user data again, did a force stop,
and cleared app data in the Android app settings.

**Comment by kistitan** [comment]
@IngoMontaigne 

For further investigation would you mind sending me your FetLife user name in an email referring to this issue?

krisztian@fetlife.com

Thanks a lot

**Comment by kistitan** [comment]
@IngoMontaigne 

May I ask if you can still reproduce the issue?
If yes, would you mind helping us out to track down this issue with installing a debug version of the App?

If you could help, please

- Download the attached zip file
- Rename it to .apk
- Copy it to your device
- Install the apk file (buy executing it from a file browser)
- After installation. please start the App
- Please use the side menu -> Log out
- Log back in
- Open your feed and / or explore page, where you cannot open the video files
- Open side menu -> Share logs.
- Send the logs to krisztian@fetlife.com

Thanks a lot
Dreamlite

[fetlife-3.0.2-beta.1+1557238580.zip](https://github.com/fetlife/android/files/3153077/fetlife-3.0.2-beta.1%2B1557238580.zip)

**Comment by IngoMontaigne** [comment]
The issue partially cleared yesterday evening. I assumed it was something
in my profile that you fixed. Most open fine, but about 10 to 20% trigger
the video player but I just get a black screen.

Would you still like me to install the troubleshooting version? Installing
it would be no problem.

**Comment by kistitan** [comment]
@IngoMontaigne 

No, in this case it does not make much sense to install it, thank you.
I checked if supporter setting is properly set, but I did not change anything on your profile, so it is a bit strange it started to work.

About the 10-20%, we got feedback that some video players do not play some of the videos. Also slow connection often make them not being able to started.

Not as a solution but a question, did you try the problematic videos with different players too?

**Comment by IngoMontaigne** [comment]
Not yet. I really only have one media player (or at least that's the only
choice i get.)
